### PR TITLE
disable edit for users and enable sysops (luckandlogicwiki)

### PR DESCRIPTION
**Pull request by Reception123**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -1198,6 +1198,11 @@
 				'createwiki' => true,
 			),
 		),
+		'+luckandlogicwiki' => array(
+			'sysop => array(
```
Comment (NDKilla) [inline LocalSettings.php]: Forgot the ' after sysop

File: LocalSettings.php [hunk 2]
```diff
@@ -1852,6 +1857,11 @@ function onPrefsPasswordAuditTestWiki( $user, $newPass, $error ) {
 	$wgGroupPermissions['user']['edit'] = false;
 }
 
+// luckandlogic.miraheze.org overrides
+if ( $wgDBname === 'luckandlogicwiki' ) {
+	$wgGroupPermissions['user']['edit'] = false;
+}
+
```
Comment (NDKilla) [inline LocalSettings.php]: Also be sure to disable anonymous editing..

**Comment by Reception123** [comment]
Why are there conflicts?

**Comment by NDKilla** [comment]
Because I merged other PRs that edit this file? Idk

**Comment by Reception123** [comment]
Conflicts to be resolved.

**Pull request closed without merging**